File: CompetitiveProgramming/DMOJ/DMPG/DMPG '19 B1 - Registration/2020-03-15/Registration.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
	int n;
	cin >> n;
	
	int sum = 0;
	for(int i = 0; i < n; i++)
	{
		int a;
		cin >> a;
		sum += a;
	}
	
	if(sum > 200)
		cout << "Over maximum capacity!" << endl;
	else
		cout << 200-sum << endl;
	
	return 0;
}

File: CompetitiveProgramming/DMOJ/DMPG/DMPG '19 B1 - Registration/2019-04-25/Registration.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Registration
{
	public static void main (String [] args) throws IOException
	{
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int n = Integer.parseInt(br.readLine());
		String [] line = br.readLine().split(" ");
		
		int count = 0;
		for(int i = 0; i < n; i++)
		{
			count += Integer.parseInt(line[i]);
		}
		
		if(count > 200)
			System.out.println("Over maximum capacity!");
		else
			System.out.println(200-count);
	}
}